docs/resources/bundle_install.md

```diff
@@ -30,6 +30,8 @@ only one can be configured at a time.
 
 - `artifactory` (Object, Sensitive) - `artifactory.username` (String) The Artifactory API username. This will likely be your hashicorp email address
 - `artifactory.token` (String) The Artifactory API token. You can sign into Artifactory and generate one
+- `artifactory.bearer_token` (String) The Artifactory Identity token - use instead of username/ API token. You can get a token by joining the
```

Comment:
I don't think this is right? We reuse token here and don't have an attribute named bearer_token in the schema.

Reply:
We'll want to remove to remove this change in `resource_bundle_install` and re-run the doc generator